scrapy/commands/bench.py

```diff
@@ -61,10 +59,10 @@ class _BenchSpider(scrapy.Spider):
     baseurl = "http://localhost:8998"
     link_extractor = LinkExtractor()
 
-    def start_requests(self) -> Iterable[Request]:
+    async def start(self) -> AsyncIterator[Any]:
```

Comment:
@wRAR I looked into the differences between [`AsyncIterable`](https://docs.python.org/3/library/collections.abc.html#collections.abc.AsyncIterable) and [`AsyncIterator`](https://docs.python.org/3/library/collections.abc.html#collections.abc.AsyncIterator), and ended up going with the later. Asynchronous generators are both, but in the end we need an iterator, and an iterable is simply an object that can provide an iterator, so asking for the iterator directly seems simpler.